x-pack/plugins/security_solution/cypress/e2e/data_sources/create_runtime_field.cy.ts

```diff
@@ -16,34 +16,43 @@ import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
 import { getNewRule } from '../../objects/rule';
 import { refreshPage } from '../../tasks/security_header';
 import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
-import { openEventsViewerFieldsBrowser } from '../../tasks/hosts/events';
-import { assertFieldDisplayed, createField } from '../../tasks/create_runtime_field';
+import { createField } from '../../tasks/create_runtime_field';
+import { openAlertsFieldBrowser } from '../../tasks/alerts';
+import { deleteRuntimeField } from '../../tasks/sourcerer';
+
+const alertRunTimeField = 'field.name.alert.page';
+const timelineRuntimeField = 'field.name.timeline';
 
 describe('Create DataView runtime field', () => {
   before(() => {
     login();
   });
 
+  before(() => {
```

Comment:
NIT: Just one before hook.

Reply:
Thanks. 🤦‍♂️ . Done now. Please find all of below changes in this commit:  https://github.com/elastic/kibana/pull/149128/commits/ad000e974eacfce9b3f02d9a69af2c5d3057a6ee